docs/contributing.rst

```diff
@@ -246,9 +245,9 @@ notes (``docs/release.rst``).
 
 The documentation can be built locally by running::
 
-    $ cd docs
-    $ make clean; make html
-    $ open _build/html/index.html
```

Comment:
How about instead of removing `open _build/html/index.html`, we add [`xdg-open`](https://linux.die.net/man/1/xdg-open) for Linux?

Reply:
I'd be minded not to include because then there would have to be multiple commands for each OS, and I'm not sure there is even one for Windows?